engine/src/flutter/lib/ui/semantics/semantics_node.h

```diff
@@ -57,6 +57,10 @@ const int kHorizontalScrollSemanticsActions =
 const int kScrollableSemanticsActions =
     kVerticalScrollSemanticsActions | kHorizontalScrollSemanticsActions;
 
+const int kSystemActions =
```

Comment:
@chunhtai What do you think of a comment like this?

```suggestion
// The following are actions that aren't user-initiated.
const int kSystemActions =
```

Reply:
Im not @chunhtai but I agree

Reply:
SGTM